scopes/react/react/webpack/transformers.ts

```diff
@@ -1,12 +1,33 @@
+import findRoot from 'find-root';
 import { WebpackConfigMutator } from '@teambit/webpack.modules.config-mutator';
 
 export function generateAddAliasesFromPeersTransformer(peers: string[]) {
   const peerAliases = peers.reduce((acc, peerName) => {
-    acc[peerName] = require.resolve(peerName);
+    const keyName = `${peerName}`;
+    // Get the dir of the package, to support cases like import react-dom/test-utils
+    // in case we just do require.resolve('react=dom') here it wil be resolved to something like
+    // node_modules/react-dom/index.js/test-utils which won't work
+    // once resolve to the folder it will resolve correctly
+    acc[keyName] = getResolvedDirOrFile(peerName);
```

Comment:
no need for keyname

```suggestion
    // Get the dir of the package, to support cases like import react-dom/test-utils
    // in case we just do require.resolve('react=dom') here it wil be resolved to something like
    // node_modules/react-dom/index.js/test-utils which won't work
    // once resolve to the folder it will resolve correctly
    acc[peerName] = getResolvedDirOrFile(peerName);
```